### ISSUE TITLE
[Suggestion] [Feature] Disable activity on Goal time reached

### ISSUE THREAD

**Issue by gpsarathy** [Razeeman/Android-SimpleTimeTracker#65]
Disabling activity button on Goal time reached on its goal cycle time. It can be used to control/limit time spent on activity we are tracking to reduce (More like habit tracker)
This should also reflect on the Widget section not just inside the app.

**Comment by Razeeman** [comment]
Hello! Do you mean that after reaching goal time activity can't be started again?

**Comment by gpsarathy** [comment]
Yes. It can be disabled until its next cycle time (Next Week - if it's a weekly goal). If it's considered as breaking user experience or deviates from main usage, similar to icon (timer like icon in current activity when goal exceeds) highlight can be added to activity notification and widget. There's a text showing goal-time, but it doesn't get updated.

Right now, there's a one-time goal notification upon completion. Nothing highlighting after that, unless you open app and see the running activity
